src/Formik.tsx

```diff
@@ -718,20 +718,36 @@ export function useFormik<Values extends FormikValues = FormikValues>({
   );
 
   const getFieldProps = React.useCallback(
-    (
-      name: string,
-      type: string
-    ): [FieldInputProps<any>, FieldMetaProps<any>] => {
-      const field = {
+    ({
+      name,
+      type,
+      value: _value,
+      as: is,
+      multiple,
+    }): [FieldInputProps<any>, FieldMetaProps<any>] => {
+      const value = getIn(state.values, name);
```

Comment:
For this `value` and `_value`, is there a name that more explicitly describes one?  Reading through it's hard to tell what the difference between each is.

Reply:
value vs. valueProp

Reply:
Ah got it, I think something like `valueState` and `valueProp` would help me differentiate.

Reply:
Yep